### PR TITLE
.new(config_mvp): complete config for mvp with all hazards and sectors we run

### PR DESCRIPTION
**Pull request by mastaia**
Note that this only works for agriculture, once the agriculture branch is merged, as it adds a new run for each crop which highly increases the computational time. 

Have not tested a complete run with it.... ;-)

### CODE REVIEW

File: run_configurations/config_mvp.py [hunk 1]
```diff
@@ -0,0 +1,433 @@
...
+"""
+import pathos as pa
+ncpus = 3
+ncpus = pa.helpers.cpu_count() - 1
+
+
+CONFIG = {
+    "run_title": "mvp_test",
+    "n_sim_years": 300,
+    "n_sim_years": 300,                     # Number of stochastic years of supply chain impacts to simulate
```
Comment (ChrisFairless) [inline run_configurations/config_mvp.py:12]: This line is here twice!

**Comment by ChrisFairless** [comment]
This looks good, thanks!!

**Comment by mastaia** [comment]
Oh, thanks , yes

**Comment by ChrisFairless** [comment]
Great, then it's good to merge 🥳

**Pull request merged**